Loop Status Extension/StatusViewController.swift

```diff
@@ -205,11 +205,24 @@ class StatusViewController: UIViewController, NCWidgetProviding {
                 if let eventualGlucose = predictedGlucose.last {
                     let formatter = NumberFormatter.glucoseFormatter(for: eventualGlucose.unit)
 
+                    var activeInsulinString = "--"
+                    if let activeInsulin = context.activeInsulin {
+                        // If the returned value doesn't indicate an error,
+                        // assign it to the label:
+                        if activeInsulin.error == "" {
```

Comment:
see my comment in `struct ActiveInsulinContext` - instead of checking for empty string, I think you should let error be an optional string value and check for nil.